pkg/util/coverage/coverage.go

```diff
@@ -78,10 +78,11 @@ func FlushCoverage() {
 	tests := []testing.InternalTest{}
 	benchmarks := []testing.InternalBenchmark{}
 	examples := []testing.InternalExample{}
+	fuzztargets := []testing.InternalFuzzTarget{}
 
 	var deps fakeTestDeps
 
-	dummyRun := testing.MainStart(deps, tests, benchmarks, examples)
+	dummyRun := testing.MainStart(deps, tests, fuzztargets, benchmarks, examples)
```

Comment:
see: https://pkg.go.dev/testing#MainStart

Reply:
despite linking to the docs, fuzztargets, and benchmarks are transposed here 🤕 https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-coverage-conformance/1522304772172943360

Reply:
https://github.com/kubernetes/kubernetes/pull/109843